src/stages/intake.ts

```diff
@@ -7,25 +7,43 @@ const log = require('debug')('wolf:s1')
 /**
  * Intake Stage (S1):
  * 
- * Takes in user generated NlpResult object and creates messageData object, stored to state.
+ * Takes in user generated NlpResult object and creates messageDataArr based on incoming NlpResults, stored to state.
  * 
  * @param nlpResult user generated NLP Object
  * 
  * @returns void
  */
 export default function intake(
   { dispatch, getState }: Store,
-  nlpResult: NlpResult,
+  nlpResults: NlpResult[],
   incomingSlotData: IncomingSlotData[],
   defaultAbility: string | null = null
 ): void {
   logState(getState())
   dispatch(startIntakeStage())
-  // MessageData derived from user nlpResult
-  const messageData: MessageData = {
-    rawText: nlpResult.message,
-    intent: nlpResult.intent,
-    entities: nlpResult.entities
+
+  // MessageDataArr derived from user nlpResults
+  // Create the messageData array
+  const messageDataArr: MessageData[] = nlpResults.map(nlpResult => {
+    return {
+      rawText: nlpResult.message,
+      intent: nlpResult.intent,
+      entities: nlpResult.entities
+    }
+  })
+
+  // Default empty MessageData object
+  // This default object is utilized if the incoming nlpResults array is empty
+  let messageData: MessageData = {
+    rawText: '',
+    intent: null,
+    entities: []
+  }
+
+  // If at least one nlpResult is present, capture first element data
+  // This is temp code. Future development will require the full messageDataArr to be saved onto WolfState
```

Comment:
LGTM. Would recommend logging (for debug purposes) the fact that we're ignoring nlp items other than 1st item

Reply:
Love the recommendation. Will add the logging comment, with the future implementation of processing all of the array, we'll remove then.

Reply:
Addressed! Added logs.